include/sta/Search.hh

```diff
@@ -753,7 +753,7 @@ protected:
 			       const MinMax *min_max,
 			       const PathAnalysisPt *path_ap) = 0;
   SearchPred *pred_;
-  std::unique_ptr<TagSet> tag_cache_ = nullptr;
+  std::unique_ptr<TagSet> tag_cache_;
 };
```

Comment:
I understand the desire to use unique_ptr, but they are not used anywhere else in OpenSTA and I don't see much point in being inconsistent. shared_ptr is a more useful beast.